script/Arcueid/Day 9/09_00_ARC09_1_1.txt

```diff
@@ -31,20 +35,25 @@ With each passing day, her demands escalate further, and I feel like it's about
 [sha:1fbbcb631dfc7a630b7e48394aa08fdb08d3d61e]{
 -- Page 16, Offset 21356.
 -- 俺が怪我をしないよう、あいつなりに気遣ってくれるのは助かるし、こっちも退屈はしなかった。
-It's not like I've been hurt or anything and she's helping me out by worrying about me in her own way. There's never a dull moment with her.
+I do have to give her credit for the fact that she is,
+ in her own way, taking care to make sure that I don't get hurt.
+ That, and the fact that there's never a dull moment when she's around.
 }
 [sha:8097be6d491d199c4f7350684c6d2e51838176d6]{
 -- Page 16, Offset 21357.
 -- なので文句も不満も浮かんでこない。
-That's why I can't be upset, or dissatisfied with her.
+As a result,
+ when I try and think of complaints or grievances I have with her,
```

Comment:
try to think

as much as "try and think" is probably what I'd _say_, it's weird when its put to paper imo